app/src/main/java/ch/epfl/favo/view/tabs/addFavor/FavorRequestView.java

```diff
@@ -358,9 +435,16 @@ private void getFavorFromView(FavorStatus status) {
     String title = titleElem.getText().toString();
     String desc = descElem.getText().toString();
     FavoLocation loc = new FavoLocation(mGpsTracker.getLocation());
-    status = FavorStatus.convertTemporaryStatus(status);
-
-    Favor favor = new Favor(title, desc, userId, loc, status);
+    // FavorStatus status = FavorStatus.convertTemporaryStatus(favorStatus);
+
+    // if this is not a new favor draft, no matter inspecting a requested favor or after picking
+    // position
+    // from map view, then do not override the favor position with current user location.
```

Comment:
This is a good catch, maybe we should have a separate function in the future for creating new favors so that we aren't calling getFavorFromView() in both situations (making new favor and editing old one)